Demo/EarlGreyExample/Podfile

```diff
@@ -30,6 +30,9 @@ end
 target "EarlGreyExampleSwiftTests" do
   project 'EarlGreyExample'
 
+  # Make sure it is compatible with other pods, i.e. R.swift
+  pod 'R.swift'
```

Comment:
I'd create a new target or add it to the Carthage tests or so? Maybe run some R.swift code to it?